sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -195,6 +209,9 @@ export default {
     };
 
     onMounted(async () => {
+      sinValidStatus.value = computed(
```

Comment:
The computed property does not need another ref variable. Can you please check other places where the computed was created and used?